ColqueOlimpia/Outfitaplication/src/GeneralClasses/Clothe.java

```diff
@@ -0,0 +1,19 @@
+package GeneralClasses;
+
+import Interfaces.IClothe;
+
+public class Clothe extends Outfit {
+    String color;
```

Comment:
Don't forget to add access modifiers to your attributes. Also add the getters and setters for them.